src/main/java/com/spotinst/sdkjava/model/responses/ClusterEcsLaunchSpecApiResponse.java

```diff
@@ -0,0 +1,7 @@
+package com.spotinst.sdkjava.model.responses;
```

Comment:
Move the file under Follow the same /ocean/ecs folder structure inside the 'model/responses/' folder.